db/migrate/20190724095857_create_reviews.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReviews < ActiveRecord::Migration[5.1]
+  def change
+    create_table :reviews do |t|
+      t.references :restaurant, index: true, foreign_key: true
+      t.string :name
+      t.integer :rating
```

Comment:
maybe add a default value for calculations purposes later? also, should this be a decimal maybe?